### ISSUE TITLE
Listbox: aria-multiselectable always true

### ISSUE THREAD

**Issue by mrkshx** [primefaces/primeng#19304]
### Describe the bug

Currently the attribute aria-multiselectable is always true no matter if multiple selection is true or false.

**This issue is there since v18**

### Pull Request Link

https://github.com/primefaces/primeng/pull/19303

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://v20.primeng.org/listbox#multiple

### Environment

v20

### Angular version

20

### PrimeNG version

v20

### Steps to reproduce the behavior

- set [multiple]="false"
- check "aria-multiselectable" of ul-Element it is set to `true`

### Expected behavior

when multiple is false, aria-multiselectable should be `false`

**Comment by mehmetcetin01140** [comment]
fixed in https://github.com/primefaces/primeng/pull/19303
